partner/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+
+class Counselor(models.Model):
+    GENDER_MALE = 0
+    GENDER_FEMALE = 1
+    GENDER_CHOICES= [(GENDER_MALE, "Male"), (GENDER_FEMALE, "Female")]
+
+    name = models.CharField(max_length=50)
+    gender = models.IntegerField(choices=GENDER_CHOICES)
+    level = models.ForeignKey(Level, on_delete=models.CASCADE)
+    counsel_count = models.IntegerField(default=0)
+    introduction = models.CharField(max_length=1000)
+    is_counsel_gt_150 = models.IntegerField(default=0)
+    profile_image_url = models.URLField(max_length=200)
+    counsel_themes = models.ManyToManyField(Theme)
+    counsel_kinds = models.ManyToManyField(Counsel_Kind)
+
+    class Meta:
+        db_table = "counselors"
+
+class Level(models.Model):
+    level = models.CharFIeld(max_length=20)
```

Comment:
```suggestion
    name = models.CharFIeld(max_length=20)
```
레벨이라는건 테이블이 말해주고있습니다.

Reply:
수정반영 했습니다.